### DISCUSSION TITLE
Helm install of cert-manager is failing

### DISCUSSION THREAD

**Discussion by hrithikwins** [hubs-community/mozilla-hubs-ce-chart#2] [Q&A] [answered]
...Successfully got an update from the "jetstack" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: INSTALLATION FAILED: Unable to continue with install: CustomResourceDefinition "certificaterequests.cert-manager.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "cert-manager"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "security"

I think there is something with values.yaml , I am constantly failing at this step
```yaml
cert-manager: #defined by either the name or alias of your dependency in Chart.yaml
  enabled: true
  namespace: security
```

this is my command after deleting all the crds and adding --skip-crds (it still fails but differently)

helm install cert-manager jetstack/cert-manager \
 --namespace security \
 --set ingressShim.defaultIssuerName=letsencrypt-issuer \
 --set ingressShim.defaultIssuerKind=ClusterIssuer \
 --set installCRDs=true \
 --skip-crds

Error: INSTALLATION FAILED: Unable to continue with install: CustomResourceDefinition "orders.acme.cert-manager.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "cert-manager"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "security"

**Comment by Doginal** [comment, marked as answer]
Where is this being installed? Could you add the output for `kubectl version` and what platform? also what is the output from `kubectl get ns security`

Maybe try removing the line above the one you added. 
```
    helm install cert-manager jetstack/cert-manager \
     --namespace security \
     --set ingressShim.defaultIssuerName=letsencrypt-issuer \
     --set ingressShim.defaultIssuerKind=ClusterIssuer
```

The part in the values.yaml isn't in use from what I remember will remove it in the next update. I was messing around with having cert-manager in the same namespace and installed by the chart but seemed more trouble than manual installing and you don't lose cert-manager when you delete the chart.
